backend/index.js

```diff
@@ -1,13 +1,107 @@
 import express from 'express';
+import dotenv from "dotenv";
+// import passport from "passport";
+import axios from "axios";
```

Comment:
Could you please use https://www.npmjs.com/package/node-fetch instead of axios for fetching in Node? The rest of the codebase (separate from this repo) uses `fetch` everywhere, so it would be great to have consistency. Thank you.